src/test/java/org/alliancegenome/curation_api/ExpressionBulkUploadFmsITCase.java

```diff
@@ -88,6 +91,8 @@ public void expressionBulkUploadAllFields() throws Exception {
 			.body("results[0].expressionPattern.whereExpressed.anatomicalStructure.curie", is(anatomicalStructureTermId))
 			.body("results[0].expressionPattern.whereExpressed.anatomicalSubstructure.curie", is(anatomicalSubstructureTermId))
 			.body("results[0].expressionPattern.whereExpressed.cellularComponentTerm.curie", is(cellularComponentTermId))
+			.body("results[0].expressionPattern.whereExpressed.cellularComponentRibbonTerm.curie", is(cellularComponentRibbonTermId))
+			.body("results[0].expressionPattern.whereExpressed.cellularComponentOther", is(false))
```

Comment:
I tend to set booleans to the non-default value for initial loads - that way you can check it isn't just the default being applied - and then do an update test where it gets set to the default.  Maybe that doesn't make sense here since the value for this field depends on the presence of another field, but the tests here wouldn't catch the issue I mentioned above about specifically setting cellularComponentOther to false in updates.

Reply:
Okay, maybe it would make sense to add a separate test specifically for that situation then.